code/modules/antagonists/cult/runes.dm

```diff
@@ -132,6 +135,8 @@ structure_check() searches for nearby cultist structures required for the invoca
 						continue
 				if(L.stat)
 					continue
+				if(!istype(user, /mob/living/simple_animal/shade))
```

Comment:
```suggestion
				if(istype(user, /mob/living/simple_animal/shade))
```
continue skips the rest of the loop for that iteration and is used to make things NOT get counted as invokers here